modules/gdscript/gdscript_function.cpp

```diff
@@ -1549,7 +1549,9 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 		// function, file, line, error, explanation
 		String err_file;
 		if (p_instance)
-			err_file = p_instance->script->path;
+			if (p_instance->script->path != "") {
```

Comment:
What's the point of this check actually? If `p_instance->script->path == ""`, then `err_file` will get an empty String assigned, which is fine as it's its current value anyway.

Didn't you mean to check `p_instance->script->is_valid()` to prevent a null pointer dereference?

Reply:
Interesting part is, the bug was fixed that way. Now I am adding the `is_valid()` checks.